tests/lib/state.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+SNAPD_STATE_PATH="$SPREAD_PATH"/snapd-state
```

Comment:
Question: where does $SPREAD_PATH point?

Reply:
to /home/gopath/src/github.com/snapcore/snapd/

Reply:
This is not terribly important but we could do something like:
```
SNAPD_STATE_PATH="$SPREAD_PATH/tests/snapd-state"
```
and add a README to that directory in the tree, explaining that a state tarball is dropped here during testing.

Reply:
Sure, makes sense